src/logic/providers/foundation-info-provider.model.ts

```diff
@@ -0,0 +1,36 @@
+import { ShiftInfoModel } from "../../common-models/shift-info.model";
+import { WorkerType } from "../../common-models/worker-info.model";
+import { ArrayHelper } from "../../helpers/array.helper";
+import { ChildrenInfoProvider } from "./children-info-provider.model";
+import { ExtraWorkersInfoProvider } from "./extra-workers-info-provider.model";
+import { ShiftsProvider } from "./shifts-provider.model";
+
+export interface FoundationInfoOptions {
```

Comment:
Ja bym uzależnił to od interfesju `Sections`. 
Czyli coś w stylu:
```
interface FoundationInfoOptions extends Pick<Sections, "NurseInfo" | "BabysitterInfo"> {
  ChildrenInfo: ChildrenInfoProvider; 
   ExtraWorkersInfo: ExtraWorkersInfoProvider;
}
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys